AppCreationScripts/sample.json

```diff
@@ -1,7 +1,7 @@
 {
   "Sample": {
     "RepositoryUrl": "ms-identity-java-webapi",
-    "Title": "Integrating Azure AD into a Java web api application",
+    "Title": "This sample demonstrates a Java web app application calling a Java Web API that is secured using Azure Active Directory using the On-Behalf-Of flow",
```

Comment:
This sample demonstrates a Java web app calling a Java Web API which calls another downstream API (Microsoft Graph) using the On-Behalf-Of flow.